<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Nova empresa</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:repeat, x1:0.531, y1:1, x2:0.525584, y2:0, stop:0 rgba(0, 180, 255, 255), stop:0.482955 rgba(70, 92, 124, 255), stop:1 rgba(0, 0, 0, 246));</string>
  </property>
  <widget class="QWidget" name="formDialog">
   <widget class="QLineEdit" name="line_name">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>70</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_name">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>70</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, stop:1 rgba(0, 0, 0, 0));
color: rgb(240, 255, 255);</string>
    </property>
    <property name="text">
     <string>Nome da empresa</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_bank">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>120</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, stop:1 rgba(0, 0, 0, 0));
color: rgb(240, 255, 255);</string>
    </property>
    <property name="text">
     <string>Banco usado (Brasil, Itau)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_bank_acc">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>170</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, stop:1 rgba(0, 0, 0, 0));
color: rgb(240, 255, 255);</string>
    </property>
    <property name="text">
     <string>Cód. conta bancária</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_default_credit">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>320</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, stop:1 rgba(0, 0, 0, 0));
color: rgb(240, 255, 255);</string>
    </property>
    <property name="text">
     <string>Cód. conta devedora padrão</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_default_credit_hist">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>370</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, stop:1 rgba(0, 0, 0, 0));
color: rgb(240, 255, 255);</string>
    </property>
    <property name="text">
     <string>Histórico devedor padrão</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_default_debit">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>220</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, stop:1 rgba(0, 0, 0, 0));
color: rgb(240, 255, 255);</string>
    </property>
    <property name="text">
     <string>Cód. conta credora padrão</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_default_debit_hist">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>270</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, stop:1 rgba(0, 0, 0, 0));
color: rgb(240, 255, 255);</string>
    </property>
    <property name="text">
     <string>Histórico credor padrão</string>
    </property>
   </widget>
   <widget class="QLabel" name="window_title">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>10</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, stop:1 rgba(0, 0, 0, 0));
color: rgb(240, 255, 255);
font-size: 36,</string>
    </property>
    <property name="text">
     <string>Adicionar nova empresa</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_bank">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>120</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_bank_acc">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>170</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_default_credit_hist">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>370</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_default_debit">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>220</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_default_debit_hist">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>270</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_default_credit">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>320</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="savebutton">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>430</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color: rgba(255,255,255,200);    
color: rgb(0, 0, 0); 
font: 12pt;
font-family: &quot;Microsoft Sans Serif&quot;;
font-weight: 400;
border-radius: 7px;
}
QPushButton:hover{ 
background-color:rgba(255,255,255,255);
}</string>
    </property>
    <property name="text">
     <string>Próximo</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cancel_button">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>430</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color: rgba(255,255,255,200);    
color: rgb(0, 0, 0); 
font: 12pt;
font-family: &quot;Microsoft Sans Serif&quot;;
font-weight: 400;
border-radius: 7px;
}
QPushButton:hover{ 
background-color:rgba(255,255,255,255);
}</string>
    </property>
    <property name="text">
     <string>Cancelar</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
